xunit-viewer.js

```diff
@@ -31,7 +31,6 @@ module.exports = async (args) => {
       const outputFile = path.resolve(process.cwd(), args.output)
       fs.writeFileSync(outputFile, result)
       console.log('Written to:', logger.file(outputFile))
-      if (!args.server) process.exit(0)
```

Comment:
if (!args.server) {
 process.exit(0)
}

Reply:
I'm sorry, I fail to understand, what your intended change is. Could you please elaborate?